cpp/modules/deck.gl/core/test/lib/attribute/attribute-manager-test.cc

```diff
@@ -77,6 +77,9 @@ TEST_F(AttributeManagerTest, Update) {
   manager->add(std::make_shared<AttributeDescriptor>("attribute-one", arrow::float32(), attributeUpdater));
   manager->add(std::make_shared<AttributeDescriptor>("attribute-two", arrow::float32(), attributeUpdater));
 
+  // TODO(ilija@unfolded.ai): Update tests to use the new WebGPU interface
```

Comment:
I would expect `GPUTable` to support `num_rows()` and `num_columns()`. Reading back data is more complicated that can wait a little.